static/js/page/api/api.js

```diff
@@ -211,9 +212,38 @@ function initializeSections() {
   });
 }
 
+function handleApiPageScroll() {
+    const $scrollTopButton = $('.scroll-button-top');
+    const $backToDocsButton = $('.scroll-button-back');
+
+        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
+            $scrollTopButton.addClass('scroll-button-top_visible')
+            $backToDocsButton.addClass('scroll-button-back_visible')
+        } else {
+            $scrollTopButton.removeClass('scroll-button-top_visible')
+            $backToDocsButton.removeClass('scroll-button-back_visible')
+        }
+    $scrollTopButton.on('click', function () {
+        window.scroll({
+            top: 0,
+            left: 0,
+            behavior: 'smooth'
+        });
+    });
+}
+
+function initializeBackToDocsButton() {
+    const $backToDocsButton = $('.back-to-button');
+    $backToDocsButton.addClass('back-to-button_visible')
```

Comment:
Why it visible by js, explain me please 😞

Reply:
Because we need this button `Kotlin Docs` in tabs panel only on one page with stdlib and kotlin.test. Otherwise this button will be rendered everywhere we have such tabs.

Reply:
My we parametrize or extends it in templates?